### DISCUSSION TITLE
Autocomplete dropdown alignment

### DISCUSSION THREAD

**Discussion by Hugo-Matias** [Megabit/Blazorise#4591] [Ideas]
Not sure if this is already implemented or doable in some form but it would be useful to have an attribute to set the suggestions dropdown alignment like in standard [Bootstrap](https://getbootstrap.com/docs/5.2/components/dropdowns/#menu-alignment).

**Comment by David-Moreira** [comment]
Please open a feature request with the suggestion.
I am not sure about the feasability of this, and that's an unusual feature for an autocomplete I guess.
But we can at least take a look and consider.

**Reply by Hugo-Matias** [reply]
I agree this might be an uncommon feature, I just thought you could leverage Bootstrap's implementation but considering you also support other providers this might not be that trivial to standardize between them all. 
No worries, I just wanted to get your opinion on the subject, if more users find this and share the need for it I guess we could open a feature request but for now I would just let it go then.
